server/services/email_service.py

```
@@ -240,6 +240,52 @@ def send_welcome_email(
     return send_email_via_microservice(to_email, subject, template, data, metadata)
 
 
+def send_alert_email(
+    to_email: str,
+    recipient_name: str,
```

Ruff (0.14.1)

Unused function argument: `recipient_name`

(ARG001)